apps/cevent_frontend/src/app/create_event/_components/Steps.tsx

```diff
@@ -107,17 +111,19 @@ const Steps = ({
         createdAt: new Date(),
         address: dateLocation.location || '',
         attendeeCount: 0,
+        activities,
       };
 
       try {
         const response = await createEvent(eventData);
         if (response === null) {
-          console.error('Error creating event:', response);
+          console.error('Error creating the event:', response);
         } else {
-          router.push(routes.myEvents);
+          console.log('Event created successfully:', response);
+          router.push('/my_events');
         }
       } catch (error) {
-        console.error('Unexpected error creating event:', error);
+        console.error('Unexpected error creating the event:', error);
```

Comment:
Please delete de comments, in case you have information comments, this should not be found in the components but in methods outside the components.